test/kata2.booleanToWord.test.js

```diff
@@ -1,5 +1,8 @@
 const { booleanToWord } = require("../src");
 
 describe("booleanToWord", () => {
-  // how do we create specs again???
+  it("returns yes when passed true and returns no when passed false", () => {
+    expect(booleanToWord(true)).toBe("yes");
+    expect(booleanToWord(false)).toBe("no");
+  });
```

Comment:
I separate out these expect statements into separate tests to make it more readable + if one of the tests fails it's easier to identify what the problem if, e.g.
```
it("returns 'yes' when passed true", () => {
  expect(booleanToWord(true)).toBe("yes");
})

it("returns 'no' when passed false", () => {
  expect(booleanToWord(false)).toBe("no");
})
```